### ISSUE TITLE
can't build blockly games with "make all" or "make languages"

### ISSUE THREAD

**Issue by duzc2** [google/blockly-games#39]
![default](https://cloud.githubusercontent.com/assets/564778/17186309/4434d43c-5467-11e6-82ab-00548e2f01fa.PNG)
![1](https://cloud.githubusercontent.com/assets/564778/17187233/06c442fa-546b-11e6-9837-d440695a95de.PNG)

excuse me .
I get the error message every time I try to build languages .
I just  make all  , or  make deps languages .

**Comment by duzc2** [comment]
I found out the SoyFileSetParser class from very old version of  closure template .
this class does not appear in new version of closure.
a versions conflict ???

**Comment by duzc2** [comment]
I found this code in Makefile:
ALL_JSON = {./,index,puzzle,maze,bird,turtle,movie,pond/docs,pond,pond/tutor,pond/duck}
and "make languages" create a folder：
![2016-07-29 00-06-37](https://cloud.githubusercontent.com/assets/564778/17205378/6a042ad8-5470-11e6-98ff-e0311184962e.png)

**Comment by duzc2** [comment]
![2016-07-29 12-26-18](https://cloud.githubusercontent.com/assets/564778/17237741/a5b24a48-5587-11e6-9af5-c8a9dc2468e4.png)
it must be some problem with my build system ...

**Comment by NeilFraser** [comment]
What happens on your shell when you execute the following:
```
ls /usr/bin/{a,b}*
```
It should print all the files in that directory that start with 'a' or 'b'.

Likewise, the '{...}' notation in the rm command should delete the generated directories in each of the application sub directories.  Nothing wrong with your build system.

**Comment by duzc2** [comment]
yes, ls command works .
but the make command does not work well.
you can see the long file name in my picture.
I'v tried on Ubuntu 16.04  , centos , and mac os.
I don't know why it does not work well.

**Comment by fthuin** [comment]
I had the same problem with `make languages`

```
java -jar third-party/SoyMsgExtractor.jar --outputFile extracted_msgs.xlf --srcs appengine/template.soy,appengine/index/template.soy,appengine/puzzle/template.soy,appengine/maze/template.soy,appengine/bird/template.soy,appengine/turtle/template.soy,appengine/movie/template.soy,appengine/pond/docs/template.soy,appengine/pond/template.soy,appengine/pond/tutor/template.soy,appengine/pond/duck/template.soy
i18n/xliff_to_json.py --xlf extracted_msgs.xlf --templates appengine/template.soy,appengine/index/template.soy,appengine/puzzle/template.soy,appengine/maze/template.soy,appengine/bird/template.soy,appengine/turtle/template.soy,appengine/movie/template.soy,appengine/pond/docs/template.soy,appengine/pond/template.soy,appengine/pond/tutor/template.soy,appengine/pond/duck/template.soy
Created file: ./json/en.json
Created file: ./json/qqq.json
Created file: ./json/keys.json
Removed extracted_msgs.xlf
Creating .xlf files...
Creating .js files...
Exception in thread "main" com.google.template.soy.base.SoySyntaxException: Error opening Soy file appengine/{.: java.io.FileNotFoundException: appengine/{. (est un dossier)
	at com.google.template.soy.base.SoySyntaxException.createWithoutMetaInfo(SoySyntaxException.java:52)
	at com.google.template.soy.soyparse.SoyFileSetParser.parseSoyFileHelper(SoyFileSetParser.java:261)
	at com.google.template.soy.soyparse.SoyFileSetParser.parseWithVersions(SoyFileSetParser.java:213)
	at com.google.template.soy.soyparse.SoyFileSetParser.parse(SoyFileSetParser.java:173)
	at com.google.template.soy.SoyFileSet.compileToJsSrcFiles(SoyFileSet.java:921)
	at com.google.template.soy.SoyToJsSrcCompiler.execMain(SoyToJsSrcCompiler.java:307)
	at com.google.template.soy.SoyToJsSrcCompiler.main(SoyToJsSrcCompiler.java:243)
Traceback (most recent call last):
  File "i18n/json_to_js.py", line 252, in <module>
    main()
  File "i18n/json_to_js.py", line 239, in main
    '--srcs', args.template])
  File "/usr/lib/python2.7/subprocess.py", line 541, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['java', '-jar', 'third-party/SoyToJsSrcCompiler.jar', '--locales', 'ace,af,ar,arz,ast,az,ba,be-tarask,bg,bn,br,ca,cdo,cs,da,de,diq,el,en,es,et,eu,fa,fi,fo,fr,frr,gl,gn,hak,he,hi,hrx,hu,ia,id,is,it,ja,ka,kbd-cyrl,km,ko,ksh,ku-latn,ky,la,lb,lki,lrc,lt,lv,mg,mk,ml,mr,ms,my,mzn,nb,ne,nl,oc,pa,pl,pms,ps,pt-br,pt,ro,ru,sc,sco,sd,si,sk,sl,sr,sv,sw,ta,tcy,te,th,tr,uk,vi,zh-hans,zh-hant', '--shouldProvideRequireSoyNamespaces', '--isUsingIjData', '--messageFilePathFormat', 'appengine/{./,index,puzzle,maze,bird,turtle,movie,pond/docs,pond,pond/tutor,pond/duck}/generated/{LOCALE}.xlf', '--outputPathFormat', 'appengine/{./,index,puzzle,maze,bird,turtle,movie,pond/docs,pond,pond/tutor,pond/duck}/generated/{LOCALE}/soy.js', '--srcs', 'appengine/{./,index,puzzle,maze,bird,turtle,movie,pond/docs,pond,pond/tutor,pond/duck}/template.soy']' returned non-zero exit status 1
```


Your fix in #40 works! Thank you @duzc2

**Comment by scott987** [comment]
I had the same problem on Ubuntu 16.04
And also fixed by #40

**Comment by NeilFraser** [comment]
Several dependencies (including Blockly) changed over time, breaking the build.  Blockly Games has been updated so that everything builds properly now.

**Comment by ojgarciab** [comment]
Make execution still fails, Ubuntu 18.04. Fixed with #40 + #81 , thanks @duzc2
